test/model/filter/user/users_by_patient_id_filter_test.dart

```diff
@@ -0,0 +1,74 @@
+
+
+import 'dart:io';
+
+import 'package:icure_dart_sdk/api.dart';
+
+import '../../../util/test_utils_backend.dart';
+import 'package:uuid/uuid.dart';
+import 'package:test/test.dart';
+import 'package:uuid/uuid_util.dart';
+
+final hashedAdmin = "{R0DLKxxRDxdtpfY542gOUZbvWkfv1KWO9QOi9yvr/2c=}39a484cbf9057072623177422172e8a173bd826d68a2b12fa8e36ff94a44a0d7";
```

Comment:
Use random value that would change every time, instead of clear password. 
Also, you can create an authentication token instead of a direct passwordHash, that will have a limited validation time